### ISSUE TITLE
chrome devtools debug error

### ISSUE THREAD

**Issue by yoonzm** [RocketChat/Rocket.Chat.ReactNative#46]
What debugging tools do you use? when I open chrome devtools,console this error,please tell me what can i do.
ERROR: 
POST http://localhost:8082/create_session net::ERR_CONNECTION_REFUSED
DOMException: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'http://localhost:8082/create_session'.(…)
Realm failed to connect to the embedded debug server inside the app
this._lazyCallableModules[name] is not a function

**Comment by yoonzm** [comment]
but only is debug android

**Comment by ggazzo** [comment]
hey @yoonzm  can you check if we "solved"or "not?

**Comment by diegolmello** [comment]
@lyz19890927 Yes, it's frustrating, but we need a local database and Realm is the best option.
Reactotron helps a lot though.
We've overwritten `console.warn` to use `Reactotron.log`.
You could do the same to `console.log`.

Have you tried Reactotron?
